java2typescript-jackson/src/test/resources/java2typescript/jackson/module/WriterPreferencesTest.enumToStringLiteralType.d.ts

```diff
@@ -3,5 +3,11 @@ export type Enum =
     | "VAL2"
     | "name";
 
+export interface TestClassHasMapWithEnumKey {
+    enumKeyMap: { [key in Enum ]: string;};
+    stringKeyMap: { [key: string ]: string;};
+    intKeyMap: { [key: string ]: string;};
```

Comment:
That made sense during Your development, but I found 3 tests that expect string key in the output, so I'd still remove it